deploy/kiali/kiali_cr.yaml

```diff
@@ -447,6 +447,9 @@ spec:
 # auth: authentication settings to connect to Grafana:
 #   ca_file: The certificate authority file to use when accessing Grafana using https. An empty string means no extra
 #       certificate authority file is used. Default is an empty string.
+#   component_status: Grafana info for the Istio Component Status masthead indicator. Only if it is enabled.
+#       app_label: APP label value of the Grafana pod(s). It defaults to grafana.
```

Comment:
Is this the app label value or the app label name? I assume it is the value of the label whose name is `istio_labels.app_label_name` from here: https://github.com/kiali/kiali-operator/blob/v1.20.0/deploy/kiali/kiali_cr.yaml#L577-L586 ?
You might want to consider calling this setting something more explicit ("app_label_value") or at least add some more detail to this comment to explain what this is. I *think* (but I am not 100% sure) that "This is the value of the app label whose name is defined in the setting 'istio_labels.app_label_name' ".

Reply:
I've approached that. Tried to do a mix of your good suggestion and mine. I think it is now more specific wrt the app_label setting and label name.